banners/templatetags/banners.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from django.template.loader import render_to_string
+
+from banners.models import Banner
+
+register = template.Library()
+
+
+@register.simple_tag
+def render_active_banner(psf_pages_only=True):
+    if not psf_pages_only:
+        banner = Banner.objects.filter(active=True, psf_pages_only=psf_pages_only).first()
+    else:
+        banner = Banner.objects.filter(active=True).first()
```

Comment:
It would be nice to have a custom `QuerySet`/`Manager` that provides an `active()` method (i.e. `Banner.objects.active().first()`)